Makefile

```diff
@@ -54,19 +46,16 @@ todo:
 	grep -rnw "TODO" .
 
 lint: bootstrap $(GOPATH)/src/github.com/golang/lint/golint
-	golint `glide novendor`
+	golint ./...
 
 vet: bootstrap
-	go vet `glide novendor`
+	go vet ./...
 
 licenses: $(GOPATH)/bin/licenses
 	@licenses  $(PACKAGE_NAME) | grep $(PACKAGE_NAME)/vendor
 
 all: clean vendor test build image puppet-module
```

Comment:
It looks like the `vendor` target was removed?

What about also running `dep` as part of the build?